tagging.tf

```
@@ -17,8 +18,8 @@ resource "null_resource" "tags" {
 
   # Code found here https://docs.microsoft.com/en-us/azure/azure-resource-manager/resource-group-using-tags#code-try-22
   provisioner "local-exec" {
-    interpreter = ["/bin/bash", "-c"]
-    command = templatefile("${path.module}/script/tag.sh.tmpl",
+    interpreter = local.is_powershell ? ["PowerShell"] : ["/bin/bash", "-c"]
```

I'm far from being a powershell expert but shouldn't we use the new executable name ? 

```suggestion
    interpreter = local.is_powershell ? ["pwsh"] : ["/bin/bash", "-c"]
```

---

After discussing with @Shr3ps , it seems that I'm wrong :)

---

PowerShell version 6+ the executable is indeed named pwsh.exe.  

I was also debating adding a lower() function to validation and the locals condition so it wouldn't care whether it was PowerShell, Powershell, or powershell, but decided the formal name as set by Microsoft was probably fine as long as it was documented.

Don't have any strong feelings either way on what is expected so long as it is documented and the validation statement in variables.tf accounts for it--if you have a preference feel free to override.

---

I agree, the variable condition is good enough.